specs/tiny/register-answer-in-learning-modes.md

```
@@ -0,0 +1,65 @@
+| Field | Value |
+|-------|-------|
+| **Branch** | `develop` |
+| **Date** | 2026-04-18 |
+| **Status** | done
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)